src/Datasource/Exception/PageOutOfBoundsException.php

```diff
@@ -22,12 +22,12 @@ class PageOutOfBoundsException extends Exception
     /**
      * {@inheritDoc}
      */
-    protected $_messageTemplate = 'Page number "%s" could not be found.';
+    protected $_messageTemplate = 'Page number %s could not be found.';
 
     /**
      * Constructor
      *
-     * @param string|null $message The error message.
+     * @param array|null $message Paging info.
```

Comment:
My test case wouldn't pass if it was not allowed :)  `Cake\Core\Exception` allows `$message` to be string or array.